3rdparty/GL/khrplatform.h

```diff
@@ -0,0 +1,290 @@
+#ifndef __khrplatform_h_
```

Comment:
Why do we need this file?

Reply:
Nvm, I see glext now includes external files ironically going against its own reason for existing to wrap opengl extensions handling in a single file.

Reply:
Included by the latest glext.h